Formula/lv2.rb

```diff
@@ -64,11 +59,12 @@ def install
     rw_info = python_shebang_rewrite_info("#{libexec}/bin/python3")
     rewrite_shebang rw_info, *Dir.glob("lv2specgen/*.py")
 
-    system "python3", "./waf", "configure",
-           "--prefix=#{prefix}", "--no-plugins", "--lv2dir=#{lib}/lv2"
-    system "python3", "./waf", "build"
-    system "python3", "./waf", "install"
+    system "meson", "build", *std_meson_args, "-Dplugins=disabled", "-Dlv2dir=#{lib}/lv2"
+    system "meson", "compile", "-C", "build"
+    system "meson", "install", "-C", "build"
 
+    (libexec/"bin").install bin/"lv2specgen.py"
+    (bin/"lv2specgen.py").write_env_script(libexec/"bin/lv2specgen.py", XDG_DATA_DIRS: opt_share)
```

Comment:
```suggestion
    (bin/"lv2specgen.py").write_env_script(libexec/"bin/lv2specgen.py", XDG_DATA_DIRS: "#{opt_share}${XDG_DATA_DIRS+:${XDG_DATA_DIRS}}")
```
Maybe?

Line length limit might not like it though.

Reply:
Thanks, split to 2 lines